### PR TITLE
Side choosing ability

### PR DESCRIPTION
**Pull request by stason314**
## Description
Simple function for setting view with `UIEdgeInsets` and ability to choosing sides which will be pinned to superview. For example, it's useful when you have insets const but need to use only two of them for some view.

## Checklist

- [x] Follows code style
- [x] Has no SwiftLint warnings
- [ ] Covered by tests
- [x] All tests are passed
- [x] Access control words were checked

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,6 @@
   <img src="img/framezilla_green.png" alt="Framezilla"/>
 </p>
 
-[![Build Status](https://travis-ci.org/Otbivnoe/Framezilla.svg?branch=master)](https://travis-ci.org/Otbivnoe/Framezilla)
-[![Version](https://img.shields.io/cocoapods/v/Framezilla.svg?style=flat)](http://cocoadocs.org/docsets/Framezilla)
-[![Carthage Compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
-[![Platform](https://img.shields.io/cocoapods/p/Framezilla.svg?style=flat)](http://cocoadocs.org/docsets/Framezilla)
-![Swift 4.0](https://img.shields.io/badge/Swift-4.0-orange.svg)
-[![License](https://img.shields.io/cocoapods/l/Framezilla.svg?style=flat)](http://cocoadocs.org/docsets/Framezilla)
-
```
Comment (artemnovichkov) [inline README.md:11]: Why did you remove all badges?

File: Sources/Maker.swift [hunk 2]
```diff
@@ -565,6 +572,30 @@ public final class Maker {
...
     }
     
+    /// Creates edge relations for superview.
+    ///
+    /// - parameter insets: The insets for setting relations for superview.
+    ///
+    /// - parameter sides: The sides which will inculed from edge insets to setting relations.
+    ///
+    /// - returns: `Maker` instance for chaining relations.
+
```
Comment (artemnovichkov) [inline Sources/Maker.swift:582]: Please add an example of using to README

File: Sources/Maker.swift [hunk 3]
```diff
@@ -19,6 +19,13 @@ enum HandlerPriority: Int {
     case low
 }
 
+public enum Sides {
+    case top
+    case bottom
+    case left
+    case right
+}
```
Comment (artemnovichkov) [inline Sources/Maker.swift:27]: What's about `vertical`, `horizontal` and `all` cases? Will it be a good option?

File: Sources/Maker.swift [hunk 4]
```diff
@@ -565,6 +572,30 @@ public final class Maker {
...
+                left(inset: insets.left)
+            case .right:
+                right(inset: insets.right)
+            case .top:
+                top(inset: insets.top)
+            }
+        }
+        return self
+    }
+
```
Comment (artemnovichkov) [inline Sources/Maker.swift:598]: Please cover this function with tests

**Review by artemnovichkov — CHANGES_REQUESTED** [review]

File: Sources/Maker.swift (hunk 3 above)
Comment (Antowkos) [reply Sources/Maker.swift:27]: I can't understand how `vertical` and `horizontal` cases should work with insets. @artemnovichkov, Could you clarify it?

**Review by Antowkos — CHANGES_REQUESTED** [review]
I see a lot of commits in your pull request, which aren't yours. Please, check the correct destination branch to merge.

File: Sources/Maker.swift (hunk 3 above)
Comment (artemnovichkov) [reply Sources/Maker.swift:27]: I mean with `vertical` case it will use both left and right insets

Comment (Antowkos) [reply Sources/Maker.swift:27]: Well, it's sounds interesting

File: README.md (hunk 1 above)
Comment (stason314) [reply README.md:11]: @artemnovichkov , @Antowkos sorry my bad I think I created branch from master 😬. I will re-create pull request with there request changes fixes from correct branch.

File: Sources/Maker.swift (hunk 3 above)
Comment (stason314) [reply Sources/Maker.swift:27]: I like vertical and horizontal 👍🏻. But think `all` cases will useless because `maker` have `public func edges(insets: UIEdgeInsets) -> Maker` method which can pin all sides.

**Comment by stason314** [comment]
Sorry my bad I think I created branch from master 😬. I will re-create pull request with there request changes fixes from correct branch (develop).
